xml/System.Messaging/MessageQueue.xml

```diff
@@ -62,7 +62,7 @@
 ## Remarks  
  The Message Queuing technology allows applications running at different times to communicate across heterogeneous networks and systems that might be temporarily offline. Applications send, receive, or peek (read without removing) messages from queues. Message Queuing is an optional component of Windows 2000 and Windows NT, and must be installed separately.  
   
- The <xref:System.Messaging.MessageQueue> class is a wrapper around Message Queuing. There are multiple versions of Message Queuing, and using the <xref:System.Messaging.MessageQueue> class can result in slightly different behavior, depending on the operating system you are using. For information about specific features of each version of Message Queuing, see the topic "What's New in Message Queuing" in the Platform SDK in MSDN.  
+ The <xref:System.Messaging.MessageQueue> class is a wrapper around Message Queuing. There are multiple versions of Message Queuing, and using the <xref:System.Messaging.MessageQueue> class can result in slightly different behavior, depending on the operating system you are using. For information about specific features of each version of Message Queuing, see the topic "What's New in Message Queuing" in Microsoft Docs.  
```

Comment:
I'd suggest just deleting the entire sentence. The only message queuing docs I could find are all archived and not well organized.

```suggestion
 The <xref:System.Messaging.MessageQueue> class is a wrapper around Message Queuing. There are multiple versions of Message Queuing, and using the <xref:System.Messaging.MessageQueue> class can result in slightly different behavior, depending on the operating system you are using.
```